app/pages/project/components/ProjectNavbar/components/NavLink/NavLink.jsx

```diff
@@ -70,7 +71,7 @@ NavLink.defaultProps = {
   disabled: false,
   isExternalLink: false,
   label: '',
-  site: '',
+  onClick: () => false,
```

Comment:
I think this should actually return `true`.

Reply:
https://stackoverflow.com/questions/18971284/event-preventdefault-vs-return-false-no-jquery

Reply:
It doesn't look you'd want to cancel the event here, so I think you're right.

Reply:
Yeah, with my original code, the default action is to cancel a click event, unless an `onClick` handler is explicitly given.